### PR TITLE
Add taiko hitsound samples for the "drum" sampleset

### PR DESCRIPTION
**Pull request by frenzibyte**
In addition to https://github.com/ppy/osu/pull/19467. Similar to how osu!(stable) handles loading of `taiko-drum` samples when requested via lookup:
```csharp
            if (sample != -1) return sample;

            if ((source & SkinSource.Osu) == 0) return -1;

            // for whatever reason, there's no taiko overrides for drum samples.
            // use the normal ones instead of playing sweet nothing.
            if (defaultFile.StartsWith("taiko-drum"))
            {
                switch (defaultFile)
                {
                    case "taiko-drum-hitnormal":
                        defaultFile = "drum-hitfinish";
                        break;
                    case "taiko-drum-hitclap":
                        defaultFile = "drum-hitclap";
                        break;
                    case "taiko-drum-hitfinish":
                        defaultFile = "drum-hitnormal";
                        break;
                    case "taiko-drum-hitwhistle":
                        defaultFile = "drum-hitwhistle";
                        break;
                }
            }
```

But rather than complicating things and doing this programmatically, I've went ahead and copied the assets from the normal "drum" sampleset and added the `taiko` prefix to them.

Note that as far as I've checked, there are no places where those `taiko-drum` samples are played solely (they're always backed by the `DrumSampleTriggerSource` audio feedback), so this change is pretty much unnoticable, but I've added it for parity with stable anyways.

### CODE REVIEW

**Comment by peppy** [comment]
Just for text record, can you explain what you mean by "silence"?

**Comment by frenzibyte** [comment]
Silence as in attempting to play the hitsound will play nothing, because the default skin provides no asset for `taiko-drum` sampleset and stable works around it by manipulating the lookup filename as attached above.

**Comment by peppy** [comment]
I still don't get it. Can you rephrase the sentence in your opening post about silence? I feel like context is missing.

**Comment by frenzibyte** [comment]
Does it read any better now? I've just realised there is no mention of the word "silence" anywhere in the code I attached, no idea where I got it from.

**Review by peppy — APPROVED** [review]
I *think* this looks correct. Time will tell.
